src/main/java/org/aksw/simba/squirrel/analyzer/impl/State.java

```diff
@@ -0,0 +1,31 @@
+package org.aksw.simba.squirrel.analyzer.impl;
```

Comment:
Can you please move all the classes for the mime type detection into another package? They do not really fit into this package. Maybe `org.aksw.simba.squirrel.analyzer.mime` or `org.aksw.simba.squirrel.analyzer.detect` would be better choices.